src/winter_onboarding_2021/4clojure/alisha.clj

```diff
@@ -3,3 +3,5 @@
 (defn problem4 [] (list :a :b :c))
 
 (defn problem5 [] (conj '(2 3 4) 1))
+
+(defn problem6 [] (list :a :b :c) (vec '(:a :b :c)) (vector :a :b :c))
```

Comment:
Again, note the commit message could be improved. 
An example could be `Problem 6: compare vectors and lists`